samples/plain.yaml

```diff
@@ -0,0 +1,31 @@
+kind: DevWorkspace
+apiVersion: workspace.devfile.io/v1alpha2
+metadata:
+  name: plain
```

Comment:
My try to test it on RHPDS was unsuccessful: 
```
2021/04/27 08:39:30 Read DevWorkspace at /devworkspace-metadata/flattened.devworkspace.yaml
2021/04/27 08:39:30 Cloning project web-nodejs-sample to web-nodejs-sample
2021/04/27 08:39:30 failed to clone project web-nodejs-sample: failed to git clone from https://github.com/che-samples/web-nodejs-sample.git: mkdir /projects/web-nodejs-sample: permission denied
```

Reply:
Well, seems the reason I face it - I used default namespace.
After I switched to devworkspace-controller(I think any different from default), it works.

Reply:
P.S. BTW My devworkspace did not get failed in such case, please consider adding init container statuses into https://github.com/tinakurian/devworkspace-operator/blob/132acb9016ea6bac99c5416fd736dd2f265ff222/controllers/workspace/provision/deployment.go#L349
```yaml
status:
  containerStatuses:
    ...
  - image: quay.io/eclipse/che-theia:next
    imageID: ""
    lastState: {}
    name: theia-ide
    ready: false
    restartCount: 0
    started: false
    state:
      waiting:
        reason: PodInitializing
  hostIP: 10.0.189.15
  initContainerStatuses:
  - image: quay.io/eclipse/che-theia-endpoint-runtime-binary:next
    imageID: ""
    lastState: {}
    name: remote-runtime-injector
    ready: false
    restartCount: 0
    state:
      waiting:
        reason: PodInitializing
  phase: Pending
  qosClass: Burstable
  startTime: "2021-04-27T09:37:31Z"

```

Reply:
Done